evalml/pipelines/utils.py

```diff
@@ -59,7 +58,8 @@ def _get_preprocessing_components(X, y, problem_type, text_columns, estimator_cl
 
     pp_components.append(Imputer)
 
-    if text_columns:
+    text_columns = list(X.select('NaturalLanguage').columns)
```

Comment:
Is there a reason we switch from `natural_language` from earlier and `NaturalLanguage` here? It'd be nice to use the same format for consistency